src/rf/requirements/workers.txt

```diff
@@ -1,2 +1,3 @@
 GDAL==1.10.0
 numpy==1.10.1
+exifread==2.1.2
```

Comment:
> Which begs the question, why do we need a separate workers.txt requirement file? These are identical:

At this point I suppose we do not. Initially we were keeping things split out so that app wouldn't have extraneous dependencies that weren't begin used. We abandoned this because it was causing more trouble than it was worth. The only thing that having a separate requirements file now allows us to do is install dependencies with gdal in scope because the python gdal package was throwing fits without it.